libs/atqr/domain/src/lib/domain-spec/stories/judgeChallenge.disl

```diff
@@ -4,29 +4,22 @@ context Challenges
 
 //função no Challenge
 
-💬 RequestJudgement? :challengeId, accomplished: boolean
+💬 RequestJudgement? :challengeId
   rules:
     challenge.status = ChallengeStatus.Overdue
 
   sendJudgementRequestEmail? :challengeId
 
   if @challenge.judge answer
-    then @challengeJudgement = ChallengeJudgement.Judged
-    when accomplished.true
-      @challengeJudgement = challengeJudgement.Accomplished
-      sendChallengeAccomplishedEmailToPlayer? :challengeId
-    when accomplished.false
-      @challengeJudgement = challengeJudgement.Failed
-      sendChallengeFailedEmailToPlayer? :challengeId
+    @challengeJudgement = ChallengeJudgement.Judged
   else
-    wait 1 day to
+    wait x day to
     sendJudgementRequestEmail? :challengeId
     if not answer
```

Comment:
you can't have a method run over days. You'll need to react to the `ANewDayArrived` event and count the days passed there